preview/src/widget.js

```diff
@@ -90,44 +90,100 @@ window.addEventListener('DOMContentLoaded', () => {
         .ready
         .then(onLoad);
 
-    const params = new URLSearchParams(window.location.search);
+    function handleExperience(previewMode, experience) {
+        if(previewMode === 'slotDefaultContent') {
+            document.getElementById('preview-experience')
+                .closest('li')
+                .remove();
+
+            return;
+        }
+
+        if (experience === null) {
+            return;
+        }
+
+        document.getElementById('preview-experience').textContent = experience;
+    }
+
+    function handleAudience(previewMode, audience) {
+        if(previewMode === 'slotDefaultContent') {
+            document.getElementById('preview-audience').textContent = 'None';
+
+            return;
+        }
+
+        if (audience === null) {
+            return;
+        }
+
+        document.getElementById('preview-audience').textContent = audience;
+    }
+
+    function handleExperiment(previewMode, variant, experiment) {
+        if (previewMode === 'slotDefaultContent' || variant === null) {
```

Comment:
Why are you looking at the variant?

Reply:
I just kept this behavior, it was a [nested IF](https://github.com/croct-tech/plug-js/blob/master/preview/src/widget.js#L122). If no variant is received the experiment is not displayed.

Reply:
I think it'd make more sense to handle experiment and variant together, in the same function

Reply:
It is possible to have a variant and not an experiment name on the token.

You can create a preview for an experience draft with an experience without having set the name of the experiment.

The draft cannot be published yet, but you can get a preview of it, and there is no name to add to it.